stable/weave-cloud/templates/deployment.yaml

```diff
@@ -26,7 +26,7 @@ spec:
       {{- end }}
       terminationGracePeriodSeconds: 30
       containers:
-      - name: {{ .Values.agent.name }}
+      - name: agent
```

Comment:
would say to keep the existing name `weave-agent` and then remove the variable from the values.yaml and in readme if exist.

but i did not fully understand why

Reply:
The purpose of this PR is to make the container name match it's name in [the other source](https://github.com/weaveworks/launcher/blob/a3573cef5c15771e2a11f6c582a16adc2b6ed2b8/service/static/agent.yaml.in#L68).

Reply:
ok got it